packages/common/src/webrtc/Peer.ts

```diff
@@ -15,6 +15,14 @@ export default class Peer {
   private _constraints: { offerToReceiveAudio: boolean, offerToReceiveVideo: boolean }
   private _negotiating: boolean = false
 
+  resetNegotiating() {
+    this._negotiating = false
+  }
+
+  isNegotiating() {
```

Comment:
Are you using it anywhere?

Reply:
I was using it during tests and thought of leaving it there for the future. But I can delete.